content/blog/2021-05-24-radiobeitrag-deutschlandfunknova/index.de.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Der OpenBikeSensor auf Deutschlandfunk
```

Comment:
Ist "Deutschlandfunk" nicht ein Radiosender, *in* dem etwas läuft? Oder "bei"? "Auf" finde ich merkwürdig.

Reply:
Gute Frage. Ich glaube Man man sagt es "läuft im Radio" - eher "im Deutschlandfunk"?